api_tests/debug_tools/eth_transaction_trace.js

```diff
@@ -0,0 +1,48 @@
+const ethutil = require("ethereumjs-util");
```

Comment:
🔧  If we're going to commit this tool it'd be nice to have some kind of comment explaining what it does and why it's useful.

Reply:
Moved to https://gist.github.com/D4nte/cc6c8ff0f0e11380a6d30229cf36c4eb